bower.json

```diff
@@ -11,7 +11,19 @@
     "type": "git",
     "url": "git://github.com/PolymerElements/app-layout"
   },
-  "main": "app-layout.html",
+  "main": [
+    "app-layout.html",
+    "helpers/helpers.html",
+    "app-drawer/app-drawer.html",
+    "app-drawer-layout/app-drawer-layout.html",
+    "app-grid/app-grid-style.html",
+    "app-header/app-header.html",
+    "app-header-layout/app-header-layout.html",
+    "app-scroll-effects/app-scroll-effects-behavior.html",
```

Comment:
To be complete, each of the effects in app-scroll-effects/effects/, as well as app-scroll-effects.html, should be listed here